components/filter/filter-overflow-group.js

```diff
@@ -27,16 +40,46 @@ class FilterOverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 		`];
 	}
 
+	constructor() {
+		super();
+
+		this._filterIds = '';
+	}
+
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
 		this.addEventListener('d2l-filter-change', this._handleFilterChange);
+
+		if (!this.tags) return;
+
+		this._filterTags = document.createElement('d2l-filter-tags');
+		this.insertAdjacentElement('afterend', this._filterTags);
+	}
+
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		if (changedProperties.has('_filterIds') && this.tags) {
+			this._filterTags.setAttribute('filter-ids', this._filterIds.trim());
+		}
 	}
 
 	convertToOverflowItem(node) {
 		const tagName = node.tagName.toLowerCase();
-		if (tagName !== 'd2l-filter') console.warn(`d2l-filter-overflow-group: ${tagName} is invalid in this group. This group should only contain d2l-filter direct child elements.`);
-		else return createFilterItem(node);
+		if (tagName !== 'd2l-filter') {
+			console.warn(`d2l-filter-overflow-group: ${tagName} is invalid in this group. This group should only contain d2l-filter direct child elements.`);
+			return;
+		}
+
+		if (this.tags && !node.hasAttribute('data-filter-tags-subscribed')) {
+			const filterId = node.id || getUniqueId();
+			node.id = filterId;
+			this._filterIds += ` ${filterId}`;
+			node.setAttribute('data-filter-tags-subscribed', 'data-filter-tags-subscribed');
```

Comment:
I initially had the id addition happening in `firstUpdated`, but it's possible that new filters could be added, so I moved that here and added this `data-filter-tags-subscribed` so we aren't re-adding an id or scanning through `_filterIds` for the id each time this function is run.

If it's thought to be unlikely filters will be added after initial render then this can just be moved to firstUpdated.

Reply:
I think I'm missing something - how do filters that aren't an overflow item get their ids added to `this._filterIds` and hooked up to tags?

Reply:
So everything that's in the `OverflowGroupMixin` gets converted to an overflow item, and then when they're needed because of a resize they get used in the overflow filter, so this function gets run on all the filters.

Reply:
Ah ok.  So we have a copy of every filter, and either the original or the overflow version will be visible with the other having `display: none;`.  And the tags get hooked up to only the original filters (where an id is added if needed).  If there was already an `id` it doesn't add a new one, and it doesn't matter that id wasn't copied to the overflow filters because its just copying the dimensions, not the outside filter.

I said the above like a statement but it's really a question lol

Reply:
Yup that's mostly correct. We don't have a copy of every filter, just the dimensions, but yup when they're chomped they get a `display: none`. The remainder is all correct.